cnxpublishing/collation.py

```diff
@@ -23,7 +23,7 @@ def collate(binder, publisher, message, cursor):
     persist those changes alongside the published content.
 
     """
-    collate_models(binder)
+    binder = collate_models(binder)
```

Comment:
This is the critical fix actually. Without this, the value of `binder` remains the same. The mocking of this function in the tests made it look like this was working.